includes/class-contact.php

```diff
@@ -249,4 +271,35 @@ protected function get_google_maps_locale() {
 
 	}
 
+	/**
+	 * Defer Google Maps iframes with JavaScript.
+	 *
+	 * @action wp_footer
+	 * @since  NEXT
+	 */
+	public function defer_map_iframes_js() {
+
+		if ( ! $this->defer_map_iframes ) {
+
+			return;
+
+		}
+
+		?>
+		<script type="text/javascript">
+		window.onload = ( function() {
+			var iframes = document.getElementsByTagName( 'iframe' );
```

Comment:
@fjarrett Why not getting the iframe class directly instead of all iframes? `wpcw-widget-contact-map`